src/cli/util.js

```diff
@@ -410,7 +410,7 @@ function eachFilename(context, patterns, callback) {
 
   try {
     const filePaths = globby
-      .sync(patterns, { dot: true, nodir: true })
+      .sync(patterns, { dot: true, expandDirectories: false })
```

Comment:
```suggestion
      .sync(patterns, { dot: true })
```

works for me when I try with globby@7.1.1 and globby@10.0.1

<https://github.com/prettier/prettier/pull/6639#issuecomment-542054241> indicates to me that there should be no need for `nodir: true` or `expandDirectories: false` here